packages/devtools_app/lib/src/shared/http/http_request_data.dart

```diff
@@ -55,14 +55,33 @@ class DartIOHttpRequestData extends NetworkRequest {
   HttpProfileRequestRef _request;
 
   final int wrapperId;
+  bool isOutStanding = false;
 
-  Future<void> getFullRequestData() {
-    return serviceManager.service!
-        .getHttpProfileRequest(
-          _request.isolateId,
-          _request.id.toString(),
-        )
-        .then((updated) => _request = updated);
+  final ValueNotifier<int> _updateCount = ValueNotifier<int>(0);
+
+  /// A notifier that changes when the request data, or it's response body
+  /// changes.
+  ValueListenable<int> get updateCount => _updateCount;
```

Comment:
Do we even use the value of this count? If not, consider renaming this and returning a `ValueListenable<void>` to indicate the value isn't useful.